### ISSUE TITLE
CAN bus analyzing [enhancement]

### ISSUE THREAD

**Issue by MstrGearbox** [gabonator/LA104#7]
Would it be possible to add automotive CAN bus analyzing or possible decoding?

**Comment by gabonator** [comment]
Good idea, actually I planned to make some application with MCP2515 canbus transceiver module attached to LA104, so instead of using logic probe pins, all the canbus traffic would go through MCP2515 chip which allows not only analysis of the traffic, but also allows to generate new packets. But I don't know when I will start doing some research on this topic, I was hoping that other people will start contributing to this project with their own applications and one of it could be this CAN analyser :)

**Comment by jilleb** [comment]
I got some LIN-UART transceivers on the way.. I might make a nice LIN analyzer for the LA104 😎
